### PR TITLE
Respond to messages from Range app letting it know the extension is installed

### PR DESCRIPTION
**Pull request by jyhsu**
#### What's this PR do?
Responds to Range app messages. We need this so that Range can show different UI if the extension is installed.

#### Definition of Done:

- [ ] Code is easy to understand and conforms with Prettier & eslint configs
- [ ] Incomplete code is marked with TODOs
- [ ] Code is suitably instrumented with logging and metrics
- [ ] Documentation has been updated as appropriate
- [ ] Manifest has been updated and version incremented correctly
- [ ] [OWASP Top 10](https://www.owasp.org/index.php/Top_10-2017_Top_10) have been considered

### CODE REVIEW

File: ext/background/background.js [hunk 1]
```diff
@@ -155,6 +155,11 @@ chrome.runtime.onMessage.addListener((request, _sender, sendResponse) => {
   return true;
 });
 
+// Respond to message from Range app to confirm extension is installed.
+chrome.runtime.onMessageExternal.addListener(function (request, sender, sendResponse) {
```
Comment (jmiller101) [inline ext/background/background.js:159]: Style nit: I think it's good to replace unused params with an underscore, or if there's multiple, prepend them with an underscore. See `background.js L28`

File: ext/manifest.dev.json [hunk 2]
```diff
@@ -40,6 +40,9 @@
       "128": "images/favicons/browser-action128.png"
     }
   },
+  "externally_connectable": {
```
Comment (jmiller101) [inline ext/manifest.dev.json:43]: Don't forget the staging manifest

**Review by jmiller101 — DISMISSED** [review]
Looks good, just a couple small comments!

File: ext/manifest.dev.json (hunk 2 above)
Comment (jyhsu) [reply ext/manifest.dev.json:43]: thanks!

**Review by jmiller101 — APPROVED** [review]

**Pull request merged**